install_functions/checks.sh

```diff
@@ -477,6 +477,34 @@ function checks_filebeatURL() {
     fi
 }
 
+function checks_development_source_tag() {
+    last_stage=""
+    source_branch="${source_branch}-${last_stage}"
+
+    # Check if the stage tag exists
+    status_code=$(curl -s -o /dev/null -w "%{http_code}" \
+        "https://api.github.com/repos/wazuh/wazuh-installation-assistant/git/refs/tags/$source_branch")
+
+    if [ "$status_code" -ne 200 ]; then
+        common_logger -e "Tag '$source_branch' does not exist. Using the source branch related to the Wazuh version ($wazuh_version)."
```

Comment:
Shouldn't it be a warning instead of an error?

Reply:
Done in [6ab5ad0](https://github.com/wazuh/wazuh-installation-assistant/pull/211/commits/6ab5ad0192120c446fa9be643b6e6c6573db61e1)